LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 Anurag Hazra
+Copyright (c) 2022 Anurag Hazra
```

Comment:
```suggestion
Copyright (c) 2020 Anurag Hazra
```

Reply:
Based on that reasoning this should be the preferred change (i.e. 003e5e245689905c9141129a4ff8f380a7fdfe37).